src/Symfony/Component/Routing/Tests/Fixtures/AttributeFixtures/AliasClassController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Symfony\Component\Routing\Tests\Fixtures\AttributeFixtures;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Routing\Attribute\Route;
+
+#[Route('/hello', alias: ['alias', 'completely_different_name'])]
```

Comment:
```suggestion
#[Route('/hello', aliases: ['alias', 'completely_different_name'])]
```
Also in the other tests

Reply:
Not sure here... as "alias" is singular in yaml / php configuration for now 🤷

Reply:
I suggested it for consistency with $defaults, $options, etc.

Reply:
Let's ignre my comment then :)